src/packages/identity-utils.ts

```diff
@@ -160,21 +160,12 @@ export class BlockstackDomainId {
 
 export class IdTranslator {
     public static cruxDomainToBlockstackDomain = (cruxDomain: CruxDomainId): BlockstackDomainId => {
```

Comment:
Add null check for cruxDomain.

Reply:
This applies to all method in this class

Reply:
Done